shell/platform/fuchsia/flutter/flatland_platform_view.cc

```diff
@@ -173,6 +188,14 @@ void FlatlandPlatformView::OnCreateView(ViewCallback on_view_created,
                       fuchsia::ui::composition::ChildViewStatus status) {
                     weak->OnChildViewStatus(id, status);
                   });
+
```

Comment:
It's paranoid, but can you check that the emplace was successful and then log an error + return if it fails?  The at()'s here will give UB if it fails and there is no check for that in release mode.

Alternatively, you can convert the "FML_DCHECK(weak->child_view_info_.count(content_id.value) == 0);" to either a CHECK, or an error+return


I tend to prefer an error+return at the very top of OnCreateView if possible, even before the lambda